app/models/concerns/relationship_bulk.rb

```diff
@@ -106,11 +109,8 @@ def run_update_callbacks(ids_json, extra_options_json)
         # Send report if needed
         if extra_options['action'] == 'accept'
           begin Relationship.smooch_send_report(r.id) rescue nil end
-          Relationship.replicate_status_to_children(r.source_id, whodunnit, team_id)
         end
       end
-      # Update un-matched field
-      ProjectMedia.where(id: target_ids).update_all(unmatched: 0)
```

Comment:
Why you deleted this line? Is it because now you call `turn_off_unmatched_field`?

Reply:
Yes I rely on `turn_off_unmatched_field` callback